src/libaktualizr/utilities/utils.cc

```diff
@@ -275,7 +275,8 @@ std::string Utils::genPrettyName() {
 std::string Utils::readFile(const boost::filesystem::path &filename, const bool trim) {
   boost::filesystem::path tmpFilename = filename;
   tmpFilename += ".new";
-
+  // that's kind of dangerous to include a specific use-case business logic into a generic function used by many use
+  // cases
```

Comment:
What would you prefer instead? If you see an actual problem, it should probably have a TODO associated with it.

Reply:
Just read a file in `Utils::readFile` and introduce another function that removes the '.new' file and utilizes the "clean" version of `Utils::readFile`. Probability that actual problem occurs is very low, but the checking and removing of the ".new" file is redundant/useless  in most of the places this function is called from.